webtau-http/src/test/resources/prices.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "id": "game1",
+    "prices": [15, 30, 40, 60]
+  },
+  {
+    "id": "game2",
+    "prices": [10, 30]
+  }
+]
```

Comment:
not sure if you care about missing newlines at the end of a file

Reply:
not really :), adding one